ci/builder/requirements.txt

```diff
@@ -21,6 +21,8 @@ kubernetes==25.3.0
 kubernetes-stubs==22.6.0.post1
 launchdarkly-api==11.0.0
 mypy==0.991
+networkx==3.0
+networkx-stubs==0.0.1
```

Comment:
How did you solve the dependency problem for these?

Reply:
After all, I used networkx - the pure-python module